plugin/helpers_bots.go

```diff
@@ -45,7 +114,10 @@ func (p *HelpersImpl) EnsureBot(bot *model.Bot) (retBotId string, retErr error)
 	// If the bot has already been created, there is nothing to do.
 	if botIdBytes != nil {
 		botId := string(botIdBytes)
-		return botId, nil
+		setImagesErr := p.setBotIconAndProfileImage(botId, o.ProfileImagePath, o.IconImagePath)
```

Comment:
i don't think we need to call `p.setBotIconAndProfileImage` here, we can just do it *once* after we create the bot no? Unless I'm missing something 🤔. It will clean up having to invoke this multiple times, and we'd just have to call this once after we create the bot i.e.

Reply:
Or nevermind. It's possible that this function `EnsureBot` is invoked a second time with options that sets the image paths.

In that case I would suggest to moving this bot creation logic into a helper function and invoking `p.setBotIconAndProfile(...)` when we get a valid bot id back, it would be much cleaner that way, otherwise imagine we have to add another option, we'd have to add it here, and then again below. Would become a tad bit difficult to maintain :)